bin/inference/pycbc_inference

```diff
@@ -40,6 +41,16 @@ from pycbc.inference import option_utils
 from pycbc.inference import prior
 from pycbc.inference import burn_in
 
+def tf_from_file(path, delimiter=" "):
```

Comment:
Don't put functions at the top of ``pycbc_inference``. There's an I/O module ``pycbc.io`` that you could probably make a new module and include these types of things.